packages/clerk-js/src/ui/elements/Modal.tsx

```diff
@@ -16,7 +18,7 @@ type ModalProps = React.PropsWithChildren<{
 
 export const Modal = withFloatingTree((props: ModalProps) => {
   const { handleClose, handleOpen, contentSx, containerSx } = props;
-  const { floating, isOpen, context, nodeId } = usePopover({
+  const { floating, isOpen, context, nodeId, toggle } = usePopover({
```

Comment:
Don't we need to call the `closeModal` function from Components.tsx as well? 
https://github.com/clerkinc/javascript/blob/82b70f2497b2f9b86b89a7d1ab1608fd27ee8017/packages/clerk-js/src/ui/Components.tsx#L199

Calling toggle correctly hides the modal, but the component state is not being updated, so it looks like we're using 2 different mechanisms to close the modal:
- usePopover.toggle
- componentsControl.closeModal

Reply:
I think that the user clicking outside the modal is equivalent to calling toggle, no?

Reply:
@desiprisg 
Clicking outside the modal calls `toggle` implicitly because we use the ref from the usePopover component, right?

Reply:
Yes, from what I gather.